### PR TITLE
excluding some jdk_net vthread testcases on aix for java 19

### LINKED ISSUE #3981: jdk_net_0 BlockingSocketOps times out on ppc64 aix for Java 19
[TRSS Test Output](https://trss.adoptium.net/output/test?id=632bf1b993ac60470bc854ff) - contains full thread dump

e.g.,

[2022-09-21T22:27:38.048Z] rerun:
[2022-09-21T22:27:38.048Z] cd /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/scratch && \
[2022-09-21T22:27:38.048Z] DISPLAY=unix:0 \
[2022-09-21T22:27:38.048Z] HOME=/home/jenkins \
[2022-09-21T22:27:38.048Z] LANG=en_US \
[2022-09-21T22:27:38.048Z] PATH=/bin:/usr/bin:/usr/sbin \
[2022-09-21T22:27:38.048Z] TZ=CUT0 \
[2022-09-21T22:27:38.048Z]     /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/openjdkbinary/j2sdk-image/bin/javac \
[2022-09-21T22:27:38.048Z]         -J-ea \
[2022-09-21T22:27:38.048Z]         -J-esa \
[2022-09-21T22:27:38.048Z]         -J-Xmx512m \
[2022-09-21T22:27:38.048Z]         -J-XX:-UseCompressedOops \
[2022-09-21T22:27:38.048Z]         -J-Djava.library.path=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/openjdkbinary/openjdk-test-image/jdk/jtreg/native \
[2022-09-21T22:27:38.048Z]         -J-Dtest.vm.opts='-ea -esa -Xmx512m -XX:-UseCompressedOops' \
[2022-09-21T22:27:38.048Z]         -J-Dtest.tool.vm.opts='-J-ea -J-esa -J-Xmx512m -J-XX:-UseCompressedOops' \
[2022-09-21T22:27:38.048Z]         -J-Dtest.compiler.opts= \
[2022-09-21T22:27:38.048Z]         -J-Dtest.java.opts= \
[2022-09-21T22:27:38.048Z]         -J-Dtest.jdk=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/openjdkbinary/j2sdk-image \
[2022-09-21T22:27:38.048Z]         -J-Dcompile.jdk=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/openjdkbinary/j2sdk-image \
[2022-09-21T22:27:38.048Z]         -J-Dtest.timeout.factor=8.0 \
[2022-09-21T22:27:38.048Z]         -J-Dtest.nativepath=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/openjdkbinary/openjdk-test-image/jdk/jtreg/native \
[2022-09-21T22:27:38.048Z]         -J-Dtest.root=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk \
[2022-09-21T22:27:38.048Z]         -J-Dtest.name=java/net/vthread/BlockingSocketOps.java#id0 \
[2022-09-21T22:27:38.048Z]         -J-Dtest.file=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread/BlockingSocketOps.java \
[2022-09-21T22:27:38.048Z]         -J-Dtest.src=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread \
[2022-09-21T22:27:38.048Z]         -J-Dtest.src.path=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/lib \
[2022-09-21T22:27:38.048Z]         -J-Dtest.classes=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/java/net/vthread/BlockingSocketOps_id0.d \
[2022-09-21T22:27:38.048Z]         -J-Dtest.class.path=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/java/net/vthread/BlockingSocketOps_id0.d:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/test/lib \
[2022-09-21T22:27:38.048Z]         -J-Dtest.class.path.prefix=/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/java/net/vthread/BlockingSocketOps_id0.d:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/test/lib \
[2022-09-21T22:27:38.048Z]         -J-Dtest.enable.preview=true \
[2022-09-21T22:27:38.048Z]         -d /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/java/net/vthread/BlockingSocketOps_id0.d \
[2022-09-21T22:27:38.048Z]         -sourcepath /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/lib \
[2022-09-21T22:27:38.048Z]         -classpath /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/java/net/vthread/BlockingSocketOps_id0.d:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/TKG/output_1663787279976/jdk_net_1/work/classes/test/lib:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/jvmtest/openjdk/jtreg/lib/testng.jar:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/jvmtest/openjdk/jtreg/lib/jcommander.jar:/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/jvmtest/openjdk/jtreg/lib/guice.jar \
[2022-09-21T22:27:38.048Z]         --enable-preview \
[2022-09-21T22:27:38.048Z]         --source=19 /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/jdk/java/net/vthread/BlockingSocketOps.java
[2022-09-21T22:27:38.048Z] direct:
[2022-09-21T22:27:38.048Z] Note: /home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/aqa-tests/openjdk/openjdk-jdk/test/lib/jdk/test/lib/thread/VThreadRunner.java uses preview features of Java SE 19.
[2022-09-21T22:27:38.048Z] Note: Recompile with -Xlint:preview for details.
[2022-09-21T22:27:38.048Z] 
[2022-09-21T22:27:38.048Z] ACTION: testng -- Error. Program `/home/jenkins/workspace/Test_openjdk19_hs_extended.openjdk_ppc64_aix_testList_1/openjdkbinary/j2sdk-image/bin/java' timed out (timeout set to 2400000ms, elapsed time including timeout handling was 2400506ms).
[2022-09-21T22:27:38.048Z] REASON: User specified action: run testng/othervm/timeout=300 BlockingSocketOps 
[2022-09-21T22:27:38.048Z] TIME:   2400.524 seconds
[2022-09-21T22:27:38.048Z] messages:
[2022-09-21T22:27:38.048Z] command: testng BlockingSocketOps
[2022-09-21T22:27:38.048Z] reason: User specified action: run testng/othervm/timeout=300 BlockingSocketOps 
[2022-09-21T22:27:38.048Z] Mode: othervm [/othervm specified]
[2022-09-21T22:27:38.048Z] Timeout information:
[2022-09-21T22:27:38.048Z] Running jstack on process 20906344
[2022-09-21T22:27:38.048Z] 2022-09-21 22:38:41

### PR DESCRIPTION
**Pull request by lumuchris256**
excluded some tests on aix for java 19

fixes #3981 
signed off by; lumuchris

### CODE REVIEW

**Comment by lumuchris256** [comment]
@smlambert

File: openjdk/excludes/ProblemList_openjdk19.txt [hunk 1]
```diff
@@ -204,6 +204,12 @@ java/net/InetAddress/InternalNameServiceWithHostsFileTest.java https://github.co
 java/nio/channels/DatagramChannel/SendReceiveMaxSize.java https://github.com/adoptium/aqa-tests/issues/3820 aix-all
 java/net/DatagramSocket/SendReceiveMaxSize.java https://github.com/adoptium/aqa-tests/issues/3820 aix-all
 com/sun/net/httpserver/simpleserver/StressDirListings.java https://github.com/adoptium/aqa-tests/issues/3841 aix-all
+java/net/vthread/HttpALot.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/BlockingSocketOps.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/HttpALot.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/BlockingSocketOps.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/InterruptHttp.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+jdk/net/ExtendedSocketOption/DontFragmentTest.java.DontFragmentTest https://github.com/adoptium/aqa-tests/issues/4024 aix-all
```
Comment (smlambert) [inline openjdk/excludes/ProblemList_openjdk19.txt:212]: DontFragmentTest.java is intermittently failing for a different reason than vthread unsupported, so we should not include it as part of this exclusion (which relates to #4024)
Execution failed: `main' threw exception: java.lang.UnsupportedOperationException: IPv6 not available

Comment (smlambert) [reply openjdk/excludes/ProblemList_openjdk19.txt:212]: @lumuchris256 - please remove line 212 from this PR

**Review by smlambert — CHANGES_REQUESTED** [review]
@lumuchris256 - looking for an update to this PR before approving and merging, please see the inline comments

File: openjdk/excludes/ProblemList_openjdk19.txt (hunk 1 above)
Comment (lumuchris256) [reply openjdk/excludes/ProblemList_openjdk19.txt:212]: noted

**Comment by lumuchris256** [comment]
> @lumuchris256 - looking for an update to this PR before approving and merging, please see the inline comments

i have committed the necessary change

**Review by smlambert — APPROVED** [review]
👍

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**